hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/catalog/TableOptionProperties.java

```diff
@@ -113,6 +116,18 @@ public static void createProperties(String basePath,
    */
   public static Map<String, String> loadFromProperties(String basePath, Configuration hadoopConf) {
     Path propertiesFilePath = getPropertiesFilePath(basePath);
+    return getPropsFromFile(basePath, hadoopConf, propertiesFilePath);
+  }
+
+  /**
+   * Read table options map from the given table base path.
+   */
+  public static Map<String, String> loadFromHoodiePropertieFile(String basePath, Configuration hadoopConf) {
+    Path propertiesFilePath = getHoodiePropertiesFilePath(basePath);
+    return getPropsFromFile(basePath, hadoopConf, propertiesFilePath);
+  }
```

Comment:
Hi, gentle ping :)

Reply:
Sorry, just saw it. Is it better to replace loadFromHoodiePropertieFile with StreamerUtil.getTableConfig here?